blogs/network_events/index.md

```diff
@@ -0,0 +1,202 @@
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: allow-egress-iperf
+spec:
+  egress:
+  - action: Pass
+    to:
+    - namespaces: {}
+  priority: 10
+  subject:
+    namespaces:
+      matchLabels:
+        kubernetes.io/metadata.name: iperf
+```
+
+![admin network policy events](./images/admin_policy_network_events.png)
+
+3. Observe egress firewall traffic
+
+For egress firewall monitoring, the eBPF agent can do the following:
+
+Trace egress traffic: Track outbound traffic and see whether it adheres to egress firewall rules.
+Detect unauthorized outbound connections: You can flag outbound traffic that violates egress rules.
+
+In the following example, there are three events, one for network policy, another for admin network policy and the third for egress firewall
+
+```yaml
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: default
+  namespace: iperf
+spec:
+  egress:
+  - to:
+      cidrSelector: 8.8.8.8/24
+    type: Allow
+  - to:
+      cidrSelector: 0.0.0.0/0
+    type: Deny
+```
+
+![egress firewall network events](./images/egress_fw_network_events.png)
+
+## Integration with the Network Observability stack
+
+- `eBPF Agent`:
+
+The agent manages a new `kprobe` hook through a configuration option.
+It also registers with the OVN observability library, allowing eBPF-generated events to be enriched
+into meaningful strings.
+
+- `Agent User Space`:
+
+The Agent user space handles the aggregation of events for processing.
+
+- `Flowlogs Pipeline`:
+
+Generated flows are transmitted over gRPC to the flowLogs-pipeline,
+where the flow content is enriched with Kubernetes metadata.
+
+- `Loki storage and Console-plugin`:
+
+The enriched flows are then sent to Loki storage, making them available for
+consumption by the NetObserv console plugin. The new events are displayed in the console.
+
+- `New Filtering Capability`:
+
+A filter is introduced to filter flows based on network event enrichments.
+
+![network events building blocks](./images/network_events_design.drawio.png)
+
+## Configuration
+
+- Enable the `OVNObservability` feature in the `FeatureGate` custom resource.
+
+```shell
+$ oc edit featuregates.config.openshift.io cluster
+```
+
+```yaml
+enabled:
+- name: OVNObservability
+```
+
+Note:
+ The above FeatureGate might be on by default at the time this feature becomes GA.
+
+- Enable Network events in the Network Observability `FlowCollector` custom resource.
+
+```yaml
+sampling: 1
+privileged: true
+features:
+- "NetworkEvents"
+```
+
+Note:
+
+- in the above example we set `sampling` to 1 to be able to capture all network events without missing any.
+
+- `privileged` is set to `true` because the `OVN observability` library needs to access local OVS socket and OVN databases.
+
+## Availability
+
+The networking events monitoring feature will be available as Developer Preview for OCP 4.17 z-stream and NetObserv release 1.7,
```

Comment:
I wouldn't use the future here, I guess when this blog post is published it won't be future anymore

```suggestion
The networking events monitoring feature is available as Developer Preview for OCP 4.17 z-stream and NetObserv release 1.7,
```